apps/system/test/unit/activities_test.js

```diff
@@ -87,10 +99,21 @@ suite('system/Activities', function() {
       subject.handleEvent({type: 'appopened'});
       assert.ok(stub.calledOnce);
     });
+
+    test('checks default actions when new app installed', function() {
+      this.sinon.stub(subject, '_onNewAppInstalled');
+
+      subject.handleEvent({
+        type: 'install',
+        application: {}
+      });
+
+      assert.ok(subject._onNewAppInstalled.calledWith({}));
+    });
```

Comment:
dispatching an `applicationinstall` instead of calling `onNewAppInstalled` directly in the `manages the default launch for the app\'s activities` test will cover the fact that the event listener is successfully installed, so we wont need this test anymore.